WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/breakpoint/BreakpointType.java

```diff
@@ -20,11 +20,11 @@
 	@SerializedName("me")
 	METHOD,
 	
```

Comment:
The type is related to its scope. However, this actually differentiates between different types of breakpoints which also have slightly different abilities. See e.g. the subinterfaces here: http://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.jdt.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fjdt%2Fdebug%2Fcore%2FIJavaBreakpoint.html